Cargo.toml

```diff
@@ -16,9 +16,15 @@ default-target = "x86_64-pc-windows-msvc"
 targets = []
 
 [dependencies]
-libloading = "0.7"
-once_cell = "1.17"
-windows = { version = "0.44", features = ["implement", "Win32_Foundation", "Win32_Graphics_Direct3D12", "Win32_System_LibraryLoader", "Win32_Storage_FileSystem"] }
+libloading = "0.8"
+# once_cell = "1."
```

Comment:
If we don't it anymore, please remove it.

Reply:
I don't think dynamic linking at runtime is easily supported with `windows-bindgen`, which this was used for.

In either case `LazyLock` is slowly being adopted into `std`, and `Once` is already available: https://doc.rust-lang.org/std/sync/struct.Once.html